src/test/java/com/iabtcf/encoder/field/LangEncoderTest.java

```diff
@@ -0,0 +1,42 @@
+package com.iabtcf.encoder.field;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class LangEncoderTest {
+    private static final String EXPECTED_VALUE_PT = "PT";
+    private static final String EXPECTED_VALUE_EN = "EN";
+    private static final String INVALID_VALUE = "Invalid";
+    private static final String EMPTY_VALUE = "";
+    private static String VALUE_PT = "";
+    private static String VALUE_EN = "";
+
+    @Before
+    public void before() {
+        VALUE_PT = getBinaryISO31661Alpha2Code('P') + getBinaryISO31661Alpha2Code('T');
+        VALUE_EN = getBinaryISO31661Alpha2Code('E') + getBinaryISO31661Alpha2Code('N');
+    }
+
+    @Test
+    public void whenValueIsValidThenShouldReturnExpectedValue() {
+        Assert.assertEquals(EXPECTED_VALUE_PT, LangEncoder.getInstance().decode(VALUE_PT));
+        Assert.assertEquals(EXPECTED_VALUE_EN, LangEncoder.getInstance().decode(VALUE_EN));
+    }
+
+    @Test
+    public void whenValueIsInvalidShouldReturnFalse() {
+        Assert.assertNull(IntEncoder.getInstance().decode(EMPTY_VALUE));
```

Comment:
See my previous comment for BooleanEncoderTest. Same here.

Reply:
The suggested change was made.